index.html

```diff
@@ -225,68 +206,49 @@ <h1 id="stats-users">Lade...</h1>
 					<br>
 					<br>
 					<br>
-					<br>
 					<h1>Features</h1>
-					<div class="slideshow-container">
-					  	<div class="imgSlide">
-							<img class="imgSlideImg" src="./assets/images/stats_dark.png" width="450" height="550">
-					    	<p class="text">Serverstatistiken in Kanalnamen</p>
-					  	</div>
-
-					  	<div class="imgSlide">
-							<img class="imgSlideImg" src="./assets/images/ticketembed_dark.png" width="450" height="550">
-					    	<p class="text">Umfangreiches Ticketssystem</p>
-					  	</div>
-
-					  	<a class="prev" onclick="plusSlides(-1)">&#10094;</a>
-					  	<a class="next" onclick="plusSlides(1)">&#10095;</a>
+					<div class="features right">
+						<img src="./assets/images/tomato_ban.png" />
+						<p><strong><span translation="main.features.moderation.title"></span></strong><br>
+						<span translation="main.features.moderation.description"></span></p>
 					</div>
-
-					<div style="text-align: center;">
-					  	<span class="dot" onclick="currentSlide(1)"></span>
-					  	<span class="dot" onclick="currentSlide(2)"></span>
+					<div class="features left">
+						<img src="./assets/images/tomato_empty.png" />
+						<p><strong><span translation="main.features.template.title"></span></strong><br>
+						<span translation="main.features.template.description"></span></p>
+					</div>
+					<div class="features right">
+						<img src="./assets/images/tomato_empty.png" />
+						<p><strong><span translation="main.features.ticket.title"></span></strong><br>
+						<span translation="main.features.ticket.description"></span></p>
 					</div>
+					<div class="features left">
+						<img src="./assets/images/tomato_empty.png" />
+						<p><strong><span translation="main.features.stats.title"></span></strong><br>
+						<span translation="main.features.stats.description"></span></p>
+					</div>
+					<br>
+					<br>
 				</div>
 
-				<p id="author">Websiten-Design erstellt von <a href="https://bookie0.repl.co/">Bookie0</a>, Dashboard von <a href="https://booky.dev/">booky10</a> und TomatoCake</p>
+				<p id="author">Websiten-Design erstellt von <a href="https://bookie0.repl.co/">Bookie0</a> und <a href="https://discords.com/">Elliana</a>, Dashboard von <a href="https://booky.dev/">booky10</a> und <a href="/discord">TomatoCake</a></p>
```

Comment:
Maybe add translations?

Reply:
Translations aren't related to this PR - similar to the bot it's an ongoing process of adding new ones.